### DISCUSSION TITLE
Code view jumping around when scrolling. Blame view doesn't have this issue.

### DISCUSSION THREAD

**Discussion by fiveNinePlusR** [community/community#135939] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

https://github.com/user-attachments/assets/866708cc-d779-47f8-976c-bbeaed12692d

see video for example.

this only started to happened recently. Seems like whatever you're doing in the `blame view` does not have this issue. 

I turned off javascript and the issue goes away but also makes the line numbering disappear after about 200 lines and also causes the rest of the site to not work because... javascript is used heavily.

**Comment by fiveNinePlusR** [comment]
also no errors or info messages in the console when scrolling.

**Reply by LiteBrite82** [reply]
os, browser, any plugins running? does this happen in all repos?

**Reply by fiveNinePlusR** [reply]
macos, firefox, some extensions but nothing github specific. I don't have any extensions that I think would cause this. I disabled most of them and it still happens. 

I logged out and it works fine. logged out in a new container tab works fine. but logged on in a new container tab doesn't work nor does being logged in on my default container. the common denominator is logged on; all the extensions are fine. the only extension i figured could maybe do something here is ublock origin but that is disabled for this site and it works fine on a logged out account.

**Reply by fiveNinePlusR** [reply]
i can reliably, and slowly scroll to line 150 on my logged in account and it will jump far down the file. seems line 150 is the trigger for whatever reason. it will also jump up from a certain line that I have not been able to determine when scrolling from the bottom to make it so that the middle section of the code file is not accessible.

**Comment by viliRocha** [comment]
Hi, @fiveNinePlusR.

Some other people here in GitHub discussions have had similar issues, especially when scrolling quickly [here](https://github.com/googlecolab/colabtools/issues/2455) you can check it. This could be due to the editor trying to maintain focus on certain elements or code blocks.

It could also be happening because of your mouse or touchpad sensitivity. Adjusting the number of lines to scroll or checking for dust in the mouse sensor might help stabilize the scrolling behavior.

Also if you are using extensions like GitLens, they might interfere with the scrolling.

It is also possible that it is a bug in the application or browser you're using, especially if this issue started recently.

So you should definitely check for updates to ensure your code editor or browser is up to date. If you're using a web-based editor, clearing your browser's cache might resolve any temporary glitches.  If possible, you can even run the application in safe mode or a clean environment to see if the issue persists without any custom settings or extensions.

Good luck!

**Reply by fiveNinePlusR** [reply]
thanks for trying but none of that is applicable to this issue. it seems as though it's a logged in vs logged out thing and only happens when i am logged into github.

**Reply by viliRocha** [reply]
Wow, that's weird.

**Comment by LiteBrite82** [comment]
@fiveNinePlusR how does your issue compare to this one? [Page randomly jumps when scrolling on phone while logged in #131711](https://github.com/orgs/community/discussions/135939). They sound super close.

**Reply by fiveNinePlusR** [reply]
@LiteBrite82 sounds very similar... though i'm on a desktop mac and on firefox. maybe the fix will fix both? (edit: actually they do mention firefox but i'm unsure if it's firefox desktop or not.)

**Reply by LiteBrite82** [reply]
Ill flag it; good report!

**Comment by LiteBrite82** [comment]
💬 Your Product Feedback Has Been Submitted 🎉
Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩
- Your input will be carefully reviewed and cataloged by members of our product teams. 
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 

Where to look to see what's shipping 👀
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

What you can do in the meantime 💻
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐

**Comment by matthewmarion** [comment]
reopening this. im having this same issue with both safari + firefox on macos. no issues with chrome.
